src/styling/Header.css

```diff
@@ -0,0 +1,116 @@
+/* Add your custom styling here */
```

Comment:
This comment is very basic per your current acquired skill level, so this could be deleted